pages/githubandmarkdown.md

```diff
@@ -12,6 +12,8 @@ You must already have a GitHub account with verified email address.
 [Markdown Wiki](http://dynalon.github.io/mdwiki/#!quickstart.md) is the software we use for our Virtual Intern website. Markdown (MD) Wiki allows us to use Markdown on GitHub so that it is easily compatible with our current development environment. To put it simply, Markdown is a markup language that is very easy to learn and allows you to quickly format your text.
 In the screenshot below, you can see an example of Markdown syntax on the left side and the way it is rendered on the right side:
 
+In order to do step3 and step5, you should be on the directory, "username.github.io". If not, you should have errors on changing branches in step5.
```

Comment:
change from "you should be on the directory" to "you should be _in_ the directory". and "If not, you _would have errors when you change_ branches in step 5."

Also, put spaces between "Step" and the number it relates to. i.e. "step3" -> "Step 3"